index.html

```diff
@@ -57,18 +57,20 @@ <h2>Upcoming</h2>
         holidays get in the way).<br>
   <h2>About</h2>
         <p><strong>People doing strange things with electricity!</strong></p>
-        <p>Part hackathon, part geek social, <strong>Dorkbot</strong> is a relaxed, informal gathering of electronic artists, circuit designers, microcontroller programmers, retro computer scavengers, video game designers, digital archeologists, hackers, cyberpunks, and mod synth gearheads in Portland, Oregon. In many respects it is the creative, dynamic 21st century successor to the <a href="https://en.wikipedia.org/wiki/Homebrew_Computer_Club">Homebrew Computer Club</a>.</p>
+        <p>
+          Dorkbot is a relaxed, informal show-and-tell for electronic artists, circuit designers, microcontroller programmers, retro computing enthusiasts, video game designers, digital archaeologists, and mod synth gearheads in Portland, Oregon.
```

Comment:
Nitpick: We shouldn't use the broader term Dorkbot for our specific weird Portland version of the larger thing. We should stay specific and call it "DorkbotPDX".
```suggestion
          DorkbotPDX is a relaxed, informal show-and-tell for electronic artists, circuit designers, microcontroller programmers, retro computing enthusiasts, video game designers, digital archaeologists, and mod synth gearheads in Portland, Oregon.
```